### DISCUSSION TITLE
python library request:  "from urllib.parse import urlparse"  does not work

### DISCUSSION THREAD

**Discussion by bethanybeachbum** [microsoft/vscode-discussions#752] [Extension Development QnA]
Simply trying to import python libraries.
Python IntelliSense (Pylance) extension is installed
The following works perfectly in a python shell in terminal mode but not in Visual Studio Code:

import urllib
from urllib.parse import urlparse  -->  This line produces:    **ImportError: No module named parse**

Is this an error of Visual Studio Code?

**Comment by isidorn** [comment]
Hi, it does not look like you are writing a VS Code extension, and this discussion is a place for VS COde extension authors.

Thus I will go ahead and close this and I suggest you open a question on StackOverflow.
